src/main/java/frc/robot/RobotManager.java

```diff
@@ -69,7 +72,9 @@ public void teleopInit() {
 
 	@Override
 	public void robotPeriodic() {
-		TimeUtils.updateCycleTime(); // Better to be first
+		roborioCycles++; // Better to be first
+		TimeUtils.updateCycleTime(); // Better to be second
+		Logger.recordOutput("RoborioCycles", roborioCycles);
```

Comment:
bro just rm comment, of course it will be after changing the roborio cycles

Reply:
do you mind if i just remove all the comments of "better to be _"

Reply:
i dont think its neccessey

Reply:
worst case you write a comment to move something if its there

Reply:
i prefer the comments to be there, i and everyone will not rmember to "worst case you write a comment to move something if its there"

Reply:
so just a comment on the roborio++ to be first?

Reply:
and the time

Reply:
both better to be first?